CMakeLists.txt

```diff
@@ -6,7 +6,11 @@ if(NOT DEFINED CMAKE_MACOSX_RPATH)
   set(CMAKE_MACOSX_RPATH 0)
 endif()
 
-set(CMAKE_CXX_STANDARD 17)
+option(GTSAM_NO_BOOST_CPP17 "Require and use boost" ON)
```

Comment:
Is this the right place? I think maybe this flag should be defined in `HandleBoost.cmake`?

Reply:
This diff should go away. I forgot to remove it